tests/ExceptionTest.php

```diff
@@ -13,7 +13,7 @@ public function testUnspecifiedExceptionCode() {
 
     $request->error = 424242;
     try {
-      throw Exception::factory($connection, $request);
+      throw Exception::factory(new Textalk\WebshopClient\Mock\ConnectionMock, $request);
```

Comment:
With this change, the connection instance is unnecessary, and using the mocked request would be preferrable.